src/presenter/trip-point-board-presenter.js

```diff
@@ -107,6 +115,15 @@ export default class TripPointBoardPresenter{
       case UpdateType.MAJOR:
         this.#renderBoard(this.points);
         break;
+      case UpdateType.INIT:
+        if(this.#tripPointsModel.isPointsReady() && this.#offersModel.isOffersReady() && this.#destinationsModel.isDestinationsReady()){
```

Comment:
```suggestion
        if(this.#tripPointsModel.isReady && this.#offersModel.isReady && this.#destinationsModel.isReady){
```
Предлагаю поменять в моделях это на свойства и назвать покороче.
В контексте же понятно от чего это `isReady`, а так получается слишком многословно.